client/src/panels/panel_declarations/finances/auth_exp_planned_spending/auth_exp_planned_spending.yaml

```diff
@@ -40,40 +40,35 @@ gov_auth_exp_planned_spending_body:
     Over the last 5 years, the government has {{gl_tt "authorized" "AUTH"}} an average spending of **{{fmt_compact1_written gov_five_year_auth_average}}** while
     {{gl_tt "actually spending" "BUD_EXP"}} an average of **{{fmt_compact1_written gov_five_year_exp_average}}**.
     
-    In **{{last_history_year}}**, **{{fmt_compact1_written last_year_lapse_amt}}** or **{{fmt_percentage1 last_year_lapse_pct}}** of approved
-    authorities were unspent at the end of the fiscal year. On average, unused authorities represented **{{fmt_percentage1 hist_avg_tot_pct}}** of authorities.
+    In **{{last_history_year}}**, **{{fmt_compact1_written last_year_lapse_amt}}** or **{{fmt_percentage1 last_year_lapse_pct}}** of authorized
+    spendings were unused at the end of the fiscal year. On average, **{{fmt_percentage1 hist_avg_tot_pct}}** of authorized spendings go unused.
```

Comment:
Grammar suggestions:
"... of authorized spending was unused..."
"... of authorized spending goes unused."

Reply:
Generally, I don't think authorized spending should be pluralized.